radish34/api/src/baseline/core/integrations/messenger/sendMessage/processResponse.js

```diff
@@ -1,6 +1,7 @@
 export default (job, done) => {
   try {
-    done(null, { message: 'API: Process Response', jobData: job.data });
+    const { data } = job;
```

Comment:
In order for keeping the code consistent and following destructuring throughout the code, I would suggest keeping destructuring even for a single property. Still, should we change remove destructuring for a single property?

Reply:
Consistency is important but to a point. Every language feature you employ in your code you are trading off for something else. In this case the trade off is one character, the '.' is turned into
`const { data } = job;` and you're adding indirection too. To find out where `data` came from I have to look at another line now to see it comes from the `job` object. It might seem minor but applied across the whole codebase it can really add up. So yes, I would say let's *not* do object destructuring in the cases where the object is used only once.

Reply:
I get it, soon will update the PR. Thank You, @breakpointer for the explanation.

Reply:
Did the changes and pushed the commit.